cmd/validate.go

```diff
@@ -85,6 +90,7 @@ func imageArg(cmd *cobra.Command, args []string) error {
 func init() {
 	rootCmd.AddCommand(validateCmd)
 	validateCmd.PersistentFlags().String("file", "", "Path or URL of a manifest to validate against.")
+	validateCmd.PersistentFlags().String("log-file", "", "File to print log output to.")
```

Comment:
Why not use stdout? That gives more flexibility for redirection.

Reply:
I believe stdout is already consumed by the user-readable output. Writing to a separate log file allows us to get separate machine-readable output, and CI can always `cat` the file if we want the developer to see it (that's what I plan on doing).